ansible/roles/common/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Check Ansible version
+  fail:
+    msg:
+      - "Ansible {{ required_ansible_version }} is required, please run the upgrade procedure"
+      - "First you must change directory to /home/teamwire/platform/ansible"
```

Comment:
Please omit this line - if this message is shown, the user is already in the correct directory

Reply:
Removed!